index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1 class="first-header">Hello Microverse!</h1>
+</body>
+</html>
```

Comment:
I can see that you have some changes already merged to the main branch. In future projects, kindly ensure that the changes required for a milestone are contained originally in the source branch and not on the base branch.